content/docs/tasks/traffic-management/debugging-istio/index.md

```diff
@@ -0,0 +1,266 @@
+172.21.165.197     443       TCP <--+
+0.0.0.0            9090      HTTP <-+
+0.0.0.0            8060      HTTP   |
+0.0.0.0            15010     HTTP   |
+0.0.0.0            15003     HTTP   |
+0.0.0.0            15004     HTTP   |
+0.0.0.0            9093      HTTP   |   Receives outbound HTTP traffic for relevant port from listener `0.0.0.0_15001`
+0.0.0.0            15007     HTTP   |
+0.0.0.0            8080      HTTP   |
+0.0.0.0            9091      HTTP   |
+0.0.0.0            9080      HTTP   |
+0.0.0.0            80        HTTP <-+
+0.0.0.0            15001     TCP    // Receives all inbound and outbound traffic to the pod from IP tables and hands over to virtual listener
+172.30.164.190     9080      HTTP   // Receives all inbound traffic on 9080 from listener `0.0.0.0_15001`
+{{< /text >}}
+
+1. From the above summary you can see that every sidecar has a listener bound to `0.0.0.0:15001` which is where IP tables routes all inbound and outbound pod traffic to. This listener has `useOriginalDst` set to true which means it hands the request over to the listener that best matches the original destination of the request. If it can't find any matching virtual listeners it sends the request to the `BlackHoleCluster` which returns a 404.
+
+{{< text bash >}}
+$ istioctl proxy-config listeners productpage-v1-6c886ff494-7vxhs --port 15001 -o json
+{
+    "name": "virtual",
+    "address": {
+        "socketAddress": {
+            "address": "0.0.0.0",
+            "portValue": 15001
+        }
+    },
+    "filterChains": [
+        {
+            "filters": [
+                {
+                    "name": "envoy.tcp_proxy",
+                    "config": {
+                        "cluster": "BlackHoleCluster",
+                        "stat_prefix": "BlackHoleCluster"
+                    }
+                }
+            ]
+        }
+    ],
+    "useOriginalDst": true
+}
+{{< /text >}}
+
+1. Our request is an outbound HTTP request to port `9080` this means it gets handed off to the `0.0.0.0:9080` virtual listener. This listener then looks up the route configuration in its configures RDS. In this case it will be looking up route `9080` in RDS configured by Pilot (via ADS).
+{{< text bash >}}
+$ istioctl proxy-config listeners productpage-v1-6c886ff494-7vxhs -o json --address 0.0.0.0 --port 9080
+...
+"rds": {
+    "config_source": {
+        "ads": {}
+    },
+    "route_config_name": "9080"
+}
+...
+{{< /text >}}
+
+1. The `9080` route configuration only has a virtual host for each service. Our request is heading to the reviews service so Envoy will select the virtual host to which our request matches a domain. Once matched on domain Envoy looks for the first route that matches the request. In this case we don't have any advanced routing so there is only one route that matches on everything. This route tells Envoy to send the request to the `outbound|9080||reviews.default.svc.cluster.local` cluster.
+{{< text bash >}}
```

Comment:
Please add blank lines before all the {{< text >}} sequences in this doc.

Reply:
This causes the count to reset, am I missing something?